### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by tcollier**
@heidi @kristjan @jrichard

Made a couple small refactors

* Rename fixture files to use a conventional naming scheme (`_` between words)
* Alphabetize attribute names in specs
* Rename `RealPage::Action` module to `RealPage::Request`
* Move `Lease` and `PicklistItem` into `RealPage::Model` module (these are all the objects returned from `Request`s)

### CODE REVIEW

File: lib/real_page.rb [hunk 1]
```diff
@@ -1,14 +1,14 @@
-require 'real_page/action/base'
-require 'real_page/action/get_leases_by_traffic_source'
-require 'real_page/action/get_marketing_sources_by_property'
+require 'real_page/model/lease'
```
Comment (heidi) [inline lib/real_page.rb]: any reason not to keep these alphabetized?

**Comment by heidi** [comment]
LGTM

File: lib/real_page.rb (hunk 1 above)
Comment (tcollier) [inline lib/real_page.rb]: I'll alphabetize them in the next PR. Some cannot be since I have to respect certain dependencies.

Comment (kristjan) [inline lib/real_page.rb]: If A depends on B, shouldn't A `require 'b'` itself?

**Pull request merged**